substrate-api/Dockerfile

```
@@ -1,11 +1,12 @@
-FROM node:16
+FROM node:23-alpine
 
+RUN apk add bash
```

Build does not work for me without this

```suggestion
RUN apk add --no-cache bash python3 make g++ \
    && ln -sf python3 /usr/bin/python

```